File: solutions/uri/1743/1743.cpp
#include <iostream>

int main() {
    int i, plug, result = 0, conector[5];

    for (i = 0; i <= 4; i++) std::cin >> conector[i];

    for (i = 0; i <= 4; i++) {
        std::cin >> plug;
        result |= plug ^ conector[i];
    }

    std::cout << ((result) ? ("Y") : ("N")) << std::endl;

    return 0;
}

File: solutions/beecrowd/1743/1743.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

connector = list(map(int, input().split()))
plug = list(map(int, input().split()))

result = 1
for x, y in zip(connector, plug):
    result &= x ^ y

if result == 1:
    output('Y\n')
else:
    output('N\n')
